martin/src/config/args/root.rs

```diff
@@ -205,7 +205,7 @@ pub fn parse_file_args<T: crate::config::file::ConfigExtras>(
 
 #[cfg(test)]
 mod tests {
-
+    #[cfg(all(feature = "pmtiles", feature = "mbtiles", feature = "cog"))]
     use insta::assert_yaml_snapshot;
```

Comment:
might be better to inline it - i.e. call it as `insta::assert_yaml_snapshot!(...);` rather than using `use` with a complex cfg